Sources/IndexStoreDB/IndexStoreDB.swift

```diff
@@ -52,7 +53,8 @@ public final class IndexStoreDB {
     waitUntilDoneInitializing wait: Bool = false,
     readonly: Bool = false,
     enableOutOfDateFileWatching: Bool = false,
-    listenToUnitEvents: Bool = true
+    listenToUnitEvents: Bool = true,
+    prefixMappings: [String] = []
```

Comment:
I think it'd be nice to give this a proper API rather than just taking strings. Eg. `[PathMapping]` with a new `PathMapping` type.